Mathlib/Algebra/Order/Hom/Monoid.lean

```diff
@@ -114,43 +103,27 @@ structure OrderMonoidHom (α β : Type*) [Preorder α] [Preorder β] [MulOneClas
 /-- Infix notation for `OrderMonoidHom`. -/
 infixr:25 " →*o " => OrderMonoidHom
 
-section
-
-/-- `OrderMonoidHomClass F α β` states that `F` is a type of ordered monoid homomorphisms.
-
-You should also extend this typeclass when you extend `OrderMonoidHom`. -/
-@[to_additive]
-class OrderMonoidHomClass (F α β : Type*) [Preorder α] [Preorder β] [MulOneClass α] [MulOneClass β]
-  [FunLike F α β] extends MonoidHomClass F α β : Prop where
-  /-- An `OrderMonoidHom` is a monotone function. -/
-  monotone (f : F) : Monotone f
-#align order_monoid_hom_class OrderMonoidHomClass
-
-end
+#noalign order_monoid_hom_class
 
 variable [Preorder α] [Preorder β] [MulOneClass α] [MulOneClass β] [FunLike F α β]
 
-/-- Turn an element of a type `F` satisfying `OrderMonoidHomClass F α β` into an actual
-`OrderMonoidHom`. This is declared as the default coercion from `F` to `α →*o β`. -/
+/-- Turn an element of a type `F` satisfying `OrderHomClass F α β` and `MonoidHomClass F α β`
+into an actual `OrderMonoidHom`. This is declared as the default coercion from `F` to `α →*o β`. -/
 @[to_additive (attr := coe)
   "Turn an element of a type `F` satisfying `OrderAddMonoidHomClass F α β` into an actual
   `OrderAddMonoidHom`. This is declared as the default coercion from `F` to `α →+o β`."]
-def OrderMonoidHomClass.toOrderMonoidHom [OrderMonoidHomClass F α β] (f : F) : α →*o β :=
-  { (f : α →* β) with monotone' := monotone f }
+def OrderMonoidHomClass.toOrderMonoidHom [OrderHomClass F α β] [MonoidHomClass F α β] (f : F) :
```

Comment:
It's a bit weird to put this in the `OrderMonoidHomClass` namespace given that such a class doesn't exist anymore, but I don't have a better suggestion.

Reply:
How about
```suggestion
def FunLike.toOrderMonoidHom [OrderHomClass F α β] [MonoidHomClass F α β] (f : F) :
```

Reply:
Let me do it for all bundled homs at once.